README.md

```diff
@@ -44,14 +44,10 @@ Thank you for looking :) Good hunting!
 - [Stack Overflow Careers](http://careers.stackoverflow.com/jobs/remote)- Remote jobs for developers.
 - [Angel List Jobs](https://angel.co/jobs#find/f!%7B%22remote%22%3Atrue%7D) - Apply privately to thousands of the best startup jobs.
 - [No Fluff Jobs](https://nofluffjobs.com/#criteria=remote" - Features remote IT jobs in a standardized and detailed way.
-- [ITflow](http://itflow.biz/) - ITflow - The best remote IT jobs.
 - [Jobbox](https://www.jobbox.io/offers?t=&amp;s=featured&amp;r=on) - Jobbox connects the best tech candidates with the best jobs.
-- [jobsRemotely](https://jobsremotely.com/remote/jobs) - Find jobs that are not restricted to a particular location.
-- [RemoteCoder](https://remotecoder.io/)- The place to find the best remote jobs.
 - [WFH](https://www.wfh.io/) - A job board focusing on remote jobs in the technology space.
 - [Jobmote](http://jobmote.com/) - Jobmote is a directory of IT career opportunities that allow you to work from home.
 - [Jobscribe](http://jobscri.be/) - Get a daily email with remote jobs at tech startups.
-- [Remote Digest](http://remotedigest.com/) - Get weekly emails with awesome remote jobs for developers.
 - [Golang Jobs](http://www.golangprojects.com/golang-remote-jobs.html) - Remote jobs for Go/Golang developers.
 
```

Comment:
Aren't you accept only USA based developers? I remember filling a long form just to be notified in the emails that you accept only developers from USA :(